lib/Dialect/Secret/Transforms/DistributeGeneric.cpp

```diff
@@ -62,6 +76,7 @@ struct CollapseSecretlessGeneric : public OpRewritePattern<GenericOp> {
   }
 };
 
+// FIXME: move this pattern so it can be reused in canonicalization.
```

Comment:
These FIXMEs are addressed in https://github.com/j2kun/heir/pull/7, which is in my fork but I will re-open it against google/heir after this PR is merged.